netconfig/netlink.py

```diff
@@ -52,7 +52,7 @@ async def monitor_state_change(queues):
         # use the _create_connection_transport method
         loop = asyncio.get_event_loop()
 
-        if type(loop) == asyncio.unix_events._UnixSelectorEventLoop:
+        if isinstance(loop) == asyncio.unix_events._UnixSelectorEventLoop:
```

Comment:
Are you sure this is the correct way to use isinstance? What testing did you do?
https://docs.python.org/3/library/functions.html#isinstance